mkdocs.yml

```diff
@@ -60,7 +61,7 @@ nav:
   - Development:
     - Architecture and sharp bits: development/architecture.md
     - Drawing: development/draw.md
-    - Contributing: development/contributing.md
+    - Contributing: CONTRIBUTING.md
```

Comment:
I don't think this will work. The `mkdocs` build takes a relative path with respect to the `docs/` folder and this file is in the parent folder.